sktime/forecasting/compose/tests/test_pipeline.py

```diff
@@ -49,3 +52,24 @@ def compute_expected_y_pred(y_train, fh):
 
     expected = compute_expected_y_pred(y_train, fh)
     np.testing.assert_array_equal(actual, expected)
+
+    def test_skip_inverse_transform():
```

Comment:
The indent seems to be wrong here, assuming this is meant to be as a separate test